include/deal.II/base/qprojector.h

```diff
@@ -92,6 +92,9 @@ class QProjector
    * is used on face <tt>face_no</tt>. For further details, see the general
    * doc for this class.
    */
+  DEAL_II_DEPRECATED_EARLY_WITH_COMMENT(
+    "Use the version of this function which takes a combined_orientation "
+    "argument instead.")
```

Comment:
Would you mind adding the `@deprecated` part to the documentation as well?

Reply:
nice catch - I will! That should be there.